toolchain/sem_ir/type.h

```diff
@@ -87,19 +87,28 @@ class TypeStore : public Yaml::Printable<TypeStore> {
   }
 
   // Gets the value representation to use for a type. This returns an
-  // invalid type if the given type is not complete.
+  // `None` type if the given type is not complete.
   auto GetValueRepr(TypeId type_id) const -> ValueRepr {
     if (auto type_info = complete_type_info_.Lookup(type_id)) {
       return type_info.value().value_repr;
     }
     return {.kind = ValueRepr::Unknown};
   }
 
-  // Sets the value representation associated with a type.
-  auto SetValueRepr(TypeId type_id, ValueRepr value_repr) -> void {
-    CARBON_CHECK(value_repr.kind != ValueRepr::Unknown);
-    auto insert_info =
-        complete_type_info_.Insert(type_id, {.value_repr = value_repr});
+  // Gets the `CompleteTypeInfo` for a type, with an empty value if the type is
+  // not complete.
+  auto GetCompleteInfo(TypeId type_id) const -> CompleteTypeInfo {
```

Comment:
It seems a bit unusual to me to drop the middle word from the return type in this function name. I suppose the reason is that this will be called as `types().GetComplete[Type]Info(...)` anyway, so the `Type` part would seem a bit redundant?

If you're indifferent, I think I'd slightly prefer to include the `Type` in this name.

Reply:
Done.